### DISCUSSION TITLE
Can i have more then one subdomain for  three different stations

### DISCUSSION THREAD

**Discussion by henkiewie** [AzuraCast/AzuraCast#6448] [Q&A] [answered]
I have a working stable install according to the docs.
But I wonder why there s only one base URL.

Why can't I have three or more different ones.

I have three subdomains. test1.domain.com test2.domain.com test3.domain.com. each pointing to their own azuracast instance.
There is only room for one base url. there is also an option to put a station url per station but that does not do much it seems.

When i put them in my own Nging proxy manager (according to the docs) they all work except for the album art. After a few seconds the album art is pointing back to the base URL API. Which has it's own selfsigned cert. The music stream keeps playing. But the album art can not display because of the selfsigned cert I use for my own NPM (I als tried the method described to put the key.crt and ssl.crt in Azuracast) but this crt is not known to the outside world.
So maybe two problem in one.

Is there a solution for this especially the problem with API url changing?

Kind regards,
Henk

**Comment by BusterNeece** [comment, marked as answer]
@henkiewie So you have a single AzuraCast installation with three stations on it? Or you have three separate AzuraCast _installations_? The difference in terminology is important in this situation.

A single AzuraCast installation can absolutely handle being served from multiple URLs, especially if you have the "Prefer Browser URL" setting enabled on the system settings panel.

You should also likely disable the "High-Performance Now Playing API" setting, because in exchange for high performance, we store a static version of the API response with the base URL "baked in" to the response, which means it can't adapt to the current URL you're using. The regular Now Playing API (the non-high-performance one) adapts things automatically to your current URL, making it better suited for those scenarios.

**Comment by henkiewie** [comment]
Hi,

Thank for this explanation. Yes it's one azuracast installation. pointing to three different subdomains. (like three different genres).
And your tip about the High-Performance Now Playing API setting fixed the problem.

Kind regards,
Henk
